docs/cloud/concepts/flows.md

```diff
@@ -85,6 +85,10 @@ mutation {
 }
 ```
 
+::: tip Prefect version
```

Comment:
Question for @joshmeek / @zdhughes: tip or warning?

Reply:
I had the same question. My gut said `tip`, but I can't exactly quantify why I leaned that way.

Reply:
Obviously this isn't a huge deal, but I was thinking warning because it's saying "Note this won't work in some circumstances!" instead of "Here's how you can improve this"

Reply:
What if instead of a box you add a badge to the `Disable Heartbeats` title? 
```
### Disable Heartbeats <Badge text="0.8.1+"/>
```

Reply:
That works too and is more visible -- let's go with that!

Reply:
Works for me-- final change incoming.